engines/stark/ui/cursor.h

```diff
@@ -87,7 +91,9 @@ class Cursor {
 	VisualText *_mouseText;
 	CursorType _currentCursorType;
 
-	bool _fading;
+	Resources::Sound* _actionHoverSound;
```

Comment:
Formatting: (Asterisk aligns to variable, not type)

Resources::Sound *_actionHoverSound;